paddle/fluid/pybind/pybind.cc

```diff
@@ -2324,6 +2325,19 @@ All parameter, weight, gradient are variables in Paddle.
   m.def("disable_profiler", platform::DisableProfiler);
   m.def("is_profiler_enabled", platform::IsProfileEnabled);
   m.def("reset_profiler", platform::ResetProfiler);
+  m.def("register_pass", [](const std::string &pass_type, py::object callable) {
+    PADDLE_ENFORCE_EQ(
+        framework::ir::PassRegistry::Instance().Has(pass_type), false,
+        platform::errors::AlreadyExists(
+            "Pass '%s' is registered more than once.", pass_type));
```

Comment:
Add:

"Pass '%s' is registered more than once. Please use another name"

Then if users name their pass the same as Paddle pass, or find the GeneratePass has problem, they know what to change.

Reply:
Done.